crates/sui-types/src/messages.rs

```diff
@@ -536,6 +792,10 @@ impl Display for SingleTransactionKind {
             Self::Genesis(_) => {
                 writeln!(writer, "Transaction Kind: Genesis")?;
             }
+            Self::ProgrammableTransaction(p) => {
+                writeln!(writer, "Transaction Kind: Programmable")?;
```

Comment:
Nit: Most of the transactions format this with an extra space

```suggestion
                writeln!(writer, "Transaction Kind : Programmable")?;
```

I have a PR out that tweaks the output for `Genesis` and `ChangeEpoch` to match (among other things).  I'm not particularly attached to that formatting, but I'm a fan of consistency.